<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayWindow</class>
 <widget class="QMainWindow" name="PlayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlackJack</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="PictureLabel">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>801</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>420</y>
      <width>651</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="HitButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(5, 54, 17); 
border-color: 4mm ridge rgb(255, 255, 255);
color: rgb(255, 232, 161);
font: 9pt &quot;Sitka&quot;;</string>
       </property>
       <property name="text">
        <string>Hit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StandButton">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(5, 54, 17); 
border-color: 4mm ridge rgb(255, 255, 255);
color: rgb(255, 232, 161);
font: 9pt &quot;Sitka&quot;;</string>
       </property>
       <property name="text">
        <string>Stand</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="UserFirstCard">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UserSecondCard">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UserThirdCard">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UserFourCard">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UserFiveCard">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UserSixCard">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UserSeventhCard">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UserEightCard">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UserNinethCard">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="UserTenthCard">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>39</y>
      <width>641</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="PlayerTotalLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255); 
font: italic 16pt &quot;Sitka&quot;;</string>
       </property>
       <property name="text">
        <string>Your score: 0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255); 
font: italic 16pt &quot;Sitka&quot;;</string>
       </property>
       <property name="text">
        <string>Your Hand</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="DilerTotalLabel">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255); 
font: italic 16pt &quot;Sitka&quot;;</string>
       </property>
       <property name="text">
        <string>                        Dealer score: ?</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255); 
font: italic 16pt &quot;Sitka&quot;;</string>
       </property>
       <property name="text">
        <string>                        Dealers Hand</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="DilerFirstCard">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DilerSecondCard">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DilerThirdCard">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DilerFourthCard">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DilerFivethCard">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DilerSixthCard">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DilerSeventhCard">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DilerEightCard">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DilerNinethCard">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="DilerTenthCard">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>50</width>
      <height>73</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Bust">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>310</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); 
font: italic 16pt &quot;Sitka&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); 
font: italic 16pt &quot;Sitka&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="PlayAgainButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>390</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(5, 54, 17); 
border-color: 4mm ridge rgb(255, 255, 255);
color: rgb(255, 232, 161);
font: 9pt &quot;Sitka&quot;;</string>
    </property>
    <property name="text">
     <string>Play Again</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
